nbqa black 5 notebooks

MarcoGorelli opened the pull request:
xref #4095

MarcoGorelli committed:
nbqa black 5 notebooks

AlexAndorra commented:
Thanks @MarcoGorelli ! Actually, for this PR, let's try a new process, to limit the number of PRs that we have to open and merge: can you make only this one PR for that one issue, but with, say, one commit per 5 notebooks?
That way, I can review every commit so that it's manageable, but we can stay on this unique PR 👌

MarcoGorelli commented:
Hey @AlexAndorra ! I think that's a good suggestion!

We introduced a regression in nbqa 0.2.3 while trying to support `pylint`, and it's now fixed - my suggestion would be to take this PR as it is so that the nbqa version gets bumped. I'll then open a separate PR where I'll apply the formatter to 5 notebooks per commit and hopefully we can merge steps 2 and 3 of the style together before too long - does that sound good to you?

AlexAndorra commented:
Yeah ok, let's go with that then 👌
